client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@open-rights-exchange/client",
   "version": "1.0.1",
   "engines": {
-    "node": ">=7.6.0"
+    "node": ">=10.0.0"
```

Comment:
We shouldn't force a higher version of Node on the client unless we need to do this. Are you doing this intentionally? If so, what's driving it?

Reply:
We need to use WHATWG URL which works under both DOM and node structure. The native url object of node supports it for node versions higher than v10

Reply:
Since this project is a client library that will go into a developers code, we don't want to force the developer to have to use a later version of Node unless we really need to. Are you sure that WHATWG URL requires 10?

Reply:
Its just different usage of native url module depending on node version.  Added the logic to use the url module depending on the node version or DOM object.